sections/semantics-textlevel.include

```diff
@@ -2986,12 +2986,12 @@ wormhole connection.&lt;/mark&gt;&lt;/p&gt;
     </pre>
 
     <figure>
-    <img src="images/sample-bdi.png" />
+    <img src="images/sample-bdi.png" alt=""/>
```

Comment:
it is conforming to leave an image inside a figure that has a caption, without an alt attribute (if the caption conveys the imprtant info in the image) and is actually preferable (to leave off rather than alt="") as alt="" means you have a caption for no content (in terms of acc API semantics exposed).